documentation/projects.md

```diff
@@ -100,6 +103,28 @@ MODULEPATH:
 Use `bolt plan show <plan-name>` to view details and parameters for a specific plan.
 ```
 
+You can also use glob patterns to match multiple plan or task names. For
+example, to surface all tasks and plans in a project named `myproject`, you
+would use the following `bolt-project.yaml` file:
+
+```yaml
+name: myproject
+plans:
+- myproject*
```

Comment:
I think `myproject::*` is a clearer way to express this (and it avoids pulling in things from  a module named `myproject2`).